File: TwoSum.java
boolean twoSum(int arr[], int target) {
        // code here
        Set<Integer> set = new HashSet<>();
        for(int num : arr){
            int diff = target - num;
            if(set.contains(diff)){
                return true;
            }
            
            set.add(num);
        }
        
        return false;
    }

File: TwoSum.cpp
bool twoSum(vector<int>& arr, int target) {
        // code here
        std::unordered_set<int> seenNumbers;

        for (int num : arr) {
            int complement = target - num;
            if (seenNumbers.find(complement) != seenNumbers.end()) {
                return true;
            }
    
            seenNumbers.insert(num);
        }
    
        return false;
    }
